packages/templates/src/raw/tv-show-recap/template.json

```diff
@@ -0,0 +1,5335 @@
+{
+  "current": null,
+  "selection": [],
+  "story": {},
+  "version": 29,
```

Comment:
Current version is 30, make sure to run `npm run workflow:migrate` to update all templates.

Reply:
I ran the script and it updated few other files as well, also it update the version to `31`.

Should I just run `npm run workflow:migrate:templates` to update the templates only.

Reply:
@swissspidy `npm run workflow:migrate` also updates the version of text-sets, shall we just keep this PR limited to `npm run workflow:migrate:templates`?

Reply:
That works for me.

There's a GitHub actions workflow that automatically updates text sets every Monday anyway.

Reply:
perfect, I've updated the templates with migrate script.